src/clr-angular/code/syntax-highlight/_code.clarity.scss

```diff
@@ -1,6 +1,8 @@
-// Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
-// This software is released under MIT license.
-// The full license information can be found in LICENSE in the root directory of this project.
+/*!
```

Comment:
Why do all `scss` files have `/*!`?

Reply:
IntelliJ added it when I processed all of the `src`1 files. I can see if I can configure it to remove the !.